src/Model/Attributes.php

```diff
@@ -178,13 +166,23 @@ public function toArray()
      *
      * @return mixed
      */
-    public function __get($key)
+    public function &__get($key)
     {
+        if ('attributes' === $key) {
+            return $this->attributes;
+        }
+
         if ($this->mutable && $this->hasMutatorMethod($key, 'get')) {
-            return $this->{$this->buildMutatorMethod($key, 'get')}();
+            $this->mutableCache[$key] = $this->{$this->buildMutatorMethod($key, 'get')}();
+
+            return $this->mutableCache[$key];
+        }
+
+        if (!array_key_exists($key, $this->attributes)) {
+            $this->attributes[$key] = null;
```

Comment:
Why a null value is set on current instance instead of straight returning null?

Reply:
see the other answer